jobs/build/ocp4/build.groovy

```diff
@@ -476,14 +476,16 @@ def stageBuildImages() {
         def archReleaseStates = commonlib.ocp4ReleaseState[version.stream]
         // If any arch is GA, use signed for everything. See stageBuildCompose for details.
         def signing_mode = archReleaseStates['release']?'signed':'unsigned'
-
+        // TODO: make this qe additional registry parameterized?
+        buildlib.registry_quay_qe_login()
         def cmd =
             """
             ${doozerOpts}
             ${includeExclude "images", buildPlan.imagesIncluded, buildPlan.imagesExcluded}
             images:build
             --repo-type ${signing_mode}
             --push-to-defaults
```

Comment:
if it's still pushing to defaults, default is nothing... this needs to point to the QE quay location.

Reply:
or else (my preference really) configure the defaults in ocp-build-data to push to the right place.

Reply:
in my testing i tried out putting the registry and namespace in group.yml to push to -- looks like that will work great, so this is :+1: